src/libraries/Microsoft.PowerFx.Core/Microsoft.PowerFx.Core.csproj

```diff
@@ -30,11 +31,11 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<EmbeddedResource Include="..\..\strings\**\PowerFxResources.resw" Condition="'$(Configuration)' == 'Debug'">
-			<Link>Strings\%(RecursiveDir)%(Filename)%(Extension)</Link>
+		<EmbeddedResource Include="..\..\strings\PowerFxResources.*.resx" Condition="'$(Configuration)' == 'Debug'">
+			<Link>strings\%(Filename)%(Extension)</Link>
 		</EmbeddedResource>
-		<EmbeddedResource Include="..\..\strings\**\PowerFxResources.resw" Exclude="$(SolutionDir)\strings\qps-ploc\**" Condition="'$(Configuration)' == 'Release'">
-			<Link>Strings\%(RecursiveDir)%(Filename)%(Extension)</Link>
+		<EmbeddedResource Include="..\..\strings\PowerFxResources.*.resx" Exclude="..\..\strings\PowerFxResources.qps-ploc.resx" Condition="'$(Configuration)' == 'Release'">
```

Comment:
>Exclude="..\..\strings\PowerFxResources.qps-ploc.resx"  [](http://example.com/codeflow?start=68&length=55)

We can probably remove this exclusion - and remove the ploc file (it is not used and hasn't been updated since it was forked from the other repository)

Reply:
It can be done in a separate PR, though

Reply:
#309